### ISSUE TITLE
Bug: Missing M2M column result in Swagger API Docs

### ISSUE THREAD

**Issue by wingkwong** [nocodb/nocodb#347]
**Steps To Reproduce**
1. Create a table called ``TableA``
2. Create a M2M column called ``ColumnA``
3. Go to Swagger API Docs, call ``/nc/xxxxxxx/api/v1/TableA``
4. M2M column data is missing in the result

**Expected behavior**
M2M column data should be included in the result

Node: **v12.22.1**
Arch: **x64**
Platform: **linux**
Docker: **true**
Database: **pg**
ProjectOnRootDB: **false**
RootDB: **pg**
PackageVersion: **0.10.3**

**Comment by wingkwong** [comment]
**UPD:**

If you are using chrome, you can open Network tab and search ``nestedList``. The API is here. It is just missing in Swagger API Docs. It should look like 

```
/nc/xxxxxxx/api/v1/TableA/nestedList?limit=25&offset=0&where=&sort=Id&childs=&parents=&many=ColumnA
```

**Comment by pranavxc** [comment]
Fixed in `v0.10.4`
